.travis.yml

```diff
@@ -22,5 +22,6 @@ script:
   - if [ -n "$(gofmt -s -l $GO_FILES)" ]; then echo "gofmt the following files:"; gofmt -s -l $GO_FILES; exit 1; fi
   - go vet ./...
   - go test -v -race $PKGS            # Run all the tests with the race detector enabled
+  - GOOS=linux GOARCH=386 go test -v $PKGS # Run all tests against a 386 architecture
```

Comment:
is explicitly setting GOOS necessary?

Reply:
In fact, it seems it's not needed! Thanks for the heads up, @ramonza!